client/src/docs/command-docs.tsx

```diff
@@ -0,0 +1,121 @@
+}
+/*
+    Start = "Start",
+    Stop = "Stop",
+    Terraform = "Terraform",
+
+ */
+export const COMMAND_DOCS: ICommandDoc[] = [
+    {
+        title: Command.AbandonFix,
+        body:
+            <CardText>
+                While you are fixing the island by terraforming (creating land or water),
+                you can decide to abandon the choice you made and start again.
+            </CardText>,
+    },
+    {
+        title: Command.ClaimHexalot,
+        body:
+            <CardText>
+                When you have reserved a hexalot, and fixed the island to make sure it follows
+                the rules, you can claim the hexalot you reserved.
+                It then belongs to you, since it is completely unique and nobody else can claim the same one.
+            </CardText>,
+    },
+    {
+        title: Command.DiscardGenes,
+        body:
+            <CardText>
+                If you don't like what you're seeing during the evolution and you've given up hope for
+                this particular gotchi, you can discard the current set of genes and start from scratch.
+            </CardText>,
+    },
+    {
+        title: Command.Evolve,
+        body:
+            <CardText>
+                You can start evolving either from the starting position hanging up in the tower
+                at the center of your hexalot, or you can ride a bit first and then start evolving.
+                Evolving creates a group of idential galapagotchis which exist in the same space
+                at the same time, but have slightly mutated genes.
+                The slowest are forgotten and the new generation is made by mutating the genes
+                of the current crop of survivors.
+            </CardText>,
+    },
+    {
+        title: Command.Home,
+        body:
+            <CardText>
+                Wherever you are, you can go back to your home position up in the tower in the
+                center of your hexalot.
+            </CardText>,
+    },
+    {
+        title: Command.MakeLand,
+        body:
+            <CardText>
+                While terraforming, you can choose to make the current hexagonal spot into land.
+            </CardText>,
+    },
+    {
+        title: Command.MakeWater,
+        body:
+            <CardText>
+                While terraforming, you can choose to make the current hexagonal spot into water.
+            </CardText>,
+    },
+    {
+        title: Command.Plan,
+        body:
+            <CardText>
+                When you plan your journey, you zoom out and see the world from above.
+                Clicking on a new neighbor hexalot extends your journey one leg further,
+                and clicking on one that is already part of the makes the journey end there.
+            </CardText>,
+    },
+    {
+        title: Command.Ride,
+        body:
+            <CardText>
+                You can ride a gotchi, which means that it will be born from the center tower of its
```

Comment:
It's not clear why this is called "ride" then if you're just unleashing a gotchi to follow a journey

Reply:
Ok, what's a better word?

Reply:
No it's a good word. I should clarify: what's unclear is why the subsequent act you describe is called "ride". Make clear to the user that their camera will be following the gotchi